app/renderer/lib/client-frameworks/java.js

```diff
@@ -8,55 +8,46 @@ class JavaFramework extends Framework {
   }
 
   wrapWithBoilerplate (code) {
-    let host = JSON.stringify(this.host);
-    let port = JSON.stringify(this.port);
     let [pkg, cls] = (() => {
       if (this.caps.platformName) {
         switch (this.caps.platformName.toLowerCase()) {
           case 'ios': return ['ios', 'IOSDriver'];
           case 'android': return ['android', 'AndroidDriver'];
+          case 'windows': return ['windows', 'WindowsDriver'];
```

Comment:
❤️

Reply:
same APIs below are only valid for mobile platforms. Not sure how much they would make sense for gecko or safari drivers

Reply:
True, but in that case the execution of the action will also return an error. I guess the best way to handle this would be to only record an action if its execution succeeds, but I haven't looked into that yet.
IMO adding the remaining drivers here is still better, since now at least it will no longer be `UnknownDriver`.